### PR TITLE
Fix BlockManager.setCurrentBlockByChildNode() with multiple Editor.js instances

### PR DESCRIPTION
**Pull request by hata6502**
https://github.com/codex-team/editor.js/issues/1503

### CODE REVIEW

File: src/components/modules/blockManager.ts [hunk 1]
```diff
@@ -573,23 +571,23 @@ export default class BlockManager extends Module {
...
     const parentFirstLevelBlock = (childNode as HTMLElement).closest(`.${Block.CSS.wrapper}`);
 
-    if (parentFirstLevelBlock) {
-      /**
-       * Update current Block's index
-       *
-       * @type {number}
-       */
-      this.currentBlockIndex = this._blocks.nodes.indexOf(parentFirstLevelBlock as HTMLElement);
+    if (!parentFirstLevelBlock?.closest(`.${this.Editor.UI.CSS.editorWrapper}`)?.isEqualNode(this.Editor.UI.nodes.wrapper)) {
```
Comment (hata6502) [inline src/components/modules/blockManager.ts:574]: Support multiple Editor.js instances by checking the equality between `this.Editor.UI.CSS.editorWrapper` and `this.Editor.UI.nodes.wrapper`.

**Review by gohabereg — APPROVED** [review]

**Comment by robonetphy** [comment]
I used the following code.
```
<!DOCTYPE html>
<html>

<head>
    <title>Testing</title>
    <meta charset="UTF-8" />
</head>

<body>
    <div id="editorjs1"></div>
    <div id="editorjs2"></div>
    <script src="https://cdn.jsdelivr.net/npm/@editorjs/header@latest"></script><!-- Header -->
    <!-- <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script> -->
    <script src="../dist/editor.js"></script>
    <script>
        var editor1 = new EditorJS({
            holder: 'editorjs1',
            tools: {
                /**
                 * Each Tool is a Plugin. Pass them via 'class' option with necessary settings {@link docs/tools.md}
                 */
                header: {
                    class: Header,
                    inlineToolbar: ['marker', 'link'],
                    config: {
                        placeholder: 'Header'
                    },
                    shortcut: 'CMD+SHIFT+H'
                }
            },
            data: {
                blocks: [
                    {
                        type: "header",
                        data: {
                            text: "Editor.js1",
                            level: 2
                        }
                    }]
            }
        });
        var editor2 = new EditorJS({
            holder: 'editorjs2',
            tools: {
                /**
                 * Each Tool is a Plugin. Pass them via 'class' option with necessary settings {@link docs/tools.md}
                 */
                header: {
                    class: Header,
                    inlineToolbar: ['marker', 'link'],
                    config: {
                        placeholder: 'Header'
                    },
                    shortcut: 'CMD+SHIFT+H'
                }
            },
            data: {
                blocks: [
                    {
                        type: "header",
                        data: {
                            text: "Editor.js2",
                            level: 2
                        }
                    }]
            }
        });
    </script>
</body>
</html>
```
Problems:
1.Toolbar is not appearing.
2.During Selection of text getting error as follow.
`Uncaught TypeError: Cannot read property 'config' of undefined`
Video:
Video of the latest editor.js v2.19.1
Video of your PR.

**Review by robonetphy — APPROVED** [review]

**Comment by robonetphy** [comment]
To Test PR we can use the following code:
```
<!DOCTYPE html>
<html>

<head>
    <title>Testing</title>
    <meta charset="UTF-8" />
</head>

<body>
    <div id="editorjs1" style="border:red solid 1px"></div>
    <div id="editorjs2" style="border:yellow solid 1px"></div>
    <script src="https://cdn.jsdelivr.net/npm/@editorjs/header@latest"></script><!-- Header -->
    <!-- <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script> -->
    <script src="../dist/editor.js"></script>
    <script>
        var editor1 = new EditorJS({
            holder: 'editorjs1'
        });
        var editor2 = new EditorJS({
            holder: 'editorjs2'
        });
    </script>
</body>

</html>
```

**Review by khaydarov — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Comment by neSpecc** [comment]
Thanks, @hata6502

**Pull request merged**